source/macos/AdaptiveCards/AdaptiveCards/AdaptiveCard.swift

```diff
@@ -168,8 +168,9 @@ public struct InputFieldConfig {
     public let placeholderTextColor: NSColor
     public let multilineFieldInsets: NSEdgeInsets
     public let errorStateConfig: ErrorStateConfig
+    public let warningBadgeImage: NSImage?
```

Comment:
The name of this image variable should be errorBadgeImage

Reply:
👍